BioCollect/src/settings/GASettingsConstant.h

```diff
@@ -39,4 +42,13 @@
 #define BIOCOLLECT_VIEW @"generic"
 #define HUB_VIEW @"hubview"
 
+// OpenID Connect Configuration
+#define COGNITO_ENABLED false
+#define COGNITO_REGION @"ap-southeast-2"
+#define COGNITO_USER_POOL @"OOXU9GW39"
+
+// #define CLIENT_ID @"3evrluqa40pn11u2f2hqah813r" COGNITO
+#define CLIENT_ID @"oidc-mobile-auth"
+#define SCOPE @"email openid profile ala/attrs ala/roles"
```

Comment:
Looks like some are cognito scopes. will that be a problem?

Reply:
It hasn't presented any issues during testing, however, I'll update them.

Reply:
Updated 👍